src/main/java/fr/jcgay/maven/plugin/buildplan/AbstractLifecycleMojo.java

```diff
@@ -34,6 +40,10 @@
     /** Allow to specify which tasks will be used to calculate execution plan. */
     @Parameter(property = "buildplan.tasks", defaultValue = "deploy")
     private String[] tasks;
+    
+    /** Allow to specify an output file to bypass console output */
+    @Parameter(property = "buildplan.outputFile")
+    private String outputFile;
```

Comment:
The switch between: 
 - a file per module
 - a single file for all modules
will be handled by the `outputFile` parameter mapping by Maven.  

It the provided value is relative (`-Dbuildplan.outputFile=buildplan_output.txt`), we will have one file per module.  
If the provided value is absolute (`-Dbuildplan.outputFile=/tmp/buildplan_output.txt`), the same file will be used by all modules (that's why we need an `appendOutput` parameter). In this case It will be great to have a header with the module name to differentiate them.

Thx!